custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -230,6 +231,23 @@ def __init__(self, options, run_mode, target_type=None):
         self._df_for_test = None
         self._test_cases_to_run = None
 
+    @staticmethod
+    def set_labels(target_type, options):
```

Comment:
Better name would be: `resolve_labels` or `resolve_labels_by_target_type`, etc.

Reply:
thank you! I sat there for like five minutes trying to remember the word "resolve"